plugins/kubernetes-backend/api-report.md

```diff
@@ -326,7 +326,10 @@ export type KubernetesObjectTypes =
 // @alpha
 export interface KubernetesServiceLocator {
```

Comment:
Is this an input-only type? As in consumers will always provide an implementation of it, rather than call an implementation themselves.

If it's input only this isn't a breaking change, since it's fine to ignore the parameter in implementations. That is unless the consumer is required to interact with the parameters in some way, but that doesn't seem to be the case.

Reply:
It's only breaking because implementations have to update the function signature, they aren't actually required to do anything with it